item.rb

```diff
@@ -3,16 +3,16 @@
 require 'date'
 
 class Item
-  attr_accessor :genre, :author, :source, :label, :publish_date
+  attr_accessor :genre, :author, :label, :publish_date
   attr_reader :id, :archived
 
   def initialize(publish_date, archived)
     @id = SecureRandom.random_number(1000)
-    @publish_date = Date.parse(publish_date)
+    @publish_date = publish_date
```

Comment:
@ZewdieMc Nice call on identifying and correcting the date parsing issue that displays an incorrect date message.

Reply:
Thank you @Besufekad-HAZ 
It was initially giving an error, then I figured out the `Date.parse` method was not necessary.